source/docs/v3/integrating.blade.md

```diff
@@ -16,3 +16,4 @@ If you're using the [automatic mode]({{ $page->link('automatic-mode') }}) & [mul
 - [Laravel Telescope]({{ $page->link('integrations/telescope') }})
 - [Livewire]({{ $page->link('integrations/livewire') }})
 - [Laravel Sanctum]({{ $page->link('integrations/sanctum') }})
+- [Vite]({{ $page->link('integrations/vite') }})
```

Comment:
You can probably just link to the feature page from here (and other places), and get rid of the integration page. The pages are pretty much identical so we don't need them both.